src/main/resources/configCivcraft.yml

```diff
@@ -1543,6 +1545,39 @@ recipes:
       pumpkin pie:
         material: PUMPKIN_PIE
         amount: 192
+  Bind_Books:
+    type: PRODUCTION
+    name: Bind Books
+    production_time: 10s
+    input:
+      leather:
+        material: LEATHER
+        amount: 6
+      paper:
+        material: PAPER
+        amount: 18
+    output:
+      books:
+        material: BOOK
```

Comment:
I'm confused. It seems to produce [these books](http://minecraft.gamepedia.com/Book) as intended when I run it on my test server. I can successfully use them to create bookshelves and book & quill books, and they refuse to open with a right click.

I thought written books were org.bukkit.Material.WRITTEN_BOOK?